addons/danger/functions/fnc_vehicle.sqf

```diff
@@ -0,0 +1,137 @@
+#include "script_component.hpp"
+/*
+ * Author: nkenny
+ * handles vehicle brain
+ *
+ * Arguments:
+ * 0: unit doing the avaluation <OBJECT>
+ * 2: current action queue <ARRAY>
+ *
+ * Return Value:
+ * number - timeout for FSM
+ *
+ * Example:
+ * [bob, angryBob] call lambs_danger_fnc_brainReact;
+ *
+ * Public: No
+*/
+params ["_unit", ["_queue", []], ["_timeout", 5]];
+
+// commander
+if !(effectiveCommander vehicle _unit isEqualTo _unit) exitWith {_timeout};
+if (_queue isEqualTo []) exitWith {_timeout};
+
+// modify priorities ~ consider adding vehicle specific changes!
+private _priorities = _unit call FUNC(brainAdjust);
+
+// pick the most relevant danger cause
+private _priority = -1;
+private _index = -1;
+{
+    private _cause = _x select 0;
+    if ((_priorities select _cause) > _priority) then {
+        _index = _forEachIndex;
+    };
+} foreach _queue;
+
+// select cause
+private _causeArray = _queue select _index;
+_causeArray params ["_cause", "_dangerPos", "_dangerUntil", "_dangerCausedBy"];
+
+// is it an attack?
+private _vehicle = vehicle _unit;
+private _attack = _cause in [0, 2, 8, 9] && {!(side _dangerCausedBy isEqualTo side _unit)};
+
+// update information
+if (_attack && {RND(0.4)}) then {[_unit, _dangerCausedBy] call FUNC(shareInformation);};
+
+// vehicle type ~ Artillery
+private _artillery = _vehicle getVariable [QGVAR(isArtillery), getNumber (configFile >> "CfgVehicles" >> (typeOf _vehicle) >> "artilleryScanner") > 0];
```

Comment:
wasn't IsArtillery a wp variable?

Reply:
@jokoho48 
Not in fn_vehicleSuppress and the code that use it! Should we change it?